appinventor/components-ios/src/ListPicker.swift

```diff
@@ -24,9 +24,13 @@ open class ListPickerActivity: UINavigationController {
     _tableViewController.navigationItem.leftBarButtonItem = cancelButton
   }
 
+//  public required init?(coder aDecoder: NSCoder) {
+//    fatalError("init(coder:) has not been implemented")
+//  }
   public required init?(coder aDecoder: NSCoder) {
-    fatalError("init(coder:) has not been implemented")
-  }
+      super.init(coder: aDecoder)
+      showAlert(message: nil)
```

Comment:
This makes more sense to me (see my comment on DateTimePickerBase.swift.

Code style: Indentation should be +2, not +4.